src/App.jsx

```diff
@@ -1,38 +1,33 @@
-import React, { Component } from "react";
+import React, { useState, useEffect } from "react";
 import { Routes, Route } from "react-router-dom";
 import Home from "./pages/Home";
 import Menu from "./pages/Menu";
 import Header from "./components/Header/Header"; 
 
 
-class App extends Component {
-  constructor(props) {
-    super(props);
-    this.state = {
-      cartCount: 0,
+const App = () => {
+  const [cartCount, setCartCount] = useState(0);
+
+  useEffect(() => {
+    const link = document.createElement("link");
+    link.href = "https://fonts.googleapis.com/css2?family=Inter:ital,opsz,wght@0,14..32,100..900;1,14..32,100..900&display=swap";
+    link.rel = "stylesheet";
+    document.head.appendChild(link);
+
+    return () => {
+      document.head.removeChild(link);
```

Comment:
and if you leave it that way, not sure if there is a need in cleaning up that link